#
# Event-In port the message pertains to:
#
# event_in_mark_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 EVENTIN1      = 0 #  [n580, n380, g080]
	uint8 EVENTIN2      = 1 #  [n380, g080]
	uint8 EVENTIN3      = 2 #  [g080]
	uint8 EVENTIN4      = 3 #  [g080]
	uint8 EVENTIN5      = 4 #  [reserved]
	uint8 EVENTIN6      = 5 #  [reserved]
	uint8 EVENTIN7      = 6 #  [reserved]
	uint8 EVENTIN8      = 7 #  [reserved]
	uint8 INTERNAL_GNSS = 31 #  Internal GNSS Receiver PPS  [n580, n380]
